src/DynamoCore/Configuration/LuceneConfig.cs

```diff
@@ -0,0 +1,71 @@
+using Lucene.Net.QueryParsers.Classic;
+using Lucene.Net.Util;
+
+namespace Dynamo.Configuration
+{
+    /// <summary>
+    /// This class includes all settings of Lucene search
+    /// </summary>
+    internal class LuceneConfig
+    {
+        /// <summary>
+        /// Specify the Lucene.Net compatibility version
+        /// </summary>
+        public static LuceneVersion LuceneNetVersion = LuceneVersion.LUCENE_48;
+
+        /// <summary>
+        /// Default operator for Lucence query parser
+        /// </summary>
+        internal static Operator DefaultOperator = Operator.OR;
+
+        internal static float MinimumSimilarity = 0.5f;
```

Comment:
@zeusongit When you started the change, did you try other values? I tried other values and it seems the setting does not make a diff, all the similarity threshold is 2

Reply:
oh nvm, I figured it out, I think the max edits are the ones in effect now